object/activities.json

```diff
@@ -26,5 +26,19 @@
     "sourceId": null,
     "unit": "kg",
     "waste": 0
+  },
+  {
+    "categories": ["transformation"],
+    "comment": "",
+    "density": 0,
+    "displayName": "Moulage par injection",
+    "elecMJ": 0,
+    "heatMJ": 0,
+    "id": "b1177e7f-e14e-415c-9077-c7063e1ab8cd",
+    "name": "injection moulding//[RER] injection moulding",
```

Comment:
`injection moulding RER` may be enough?

Reply:
what are the pros/cons to using less words ? I like putting the exact name so there is no ambiguity

Reply:
Just in case the `injection moulding//[RER] injection moulding` suddenly become `injection moulding {RER} injection moulding` which does happen (changes in brightway, strategies, db version, etc.)